giskard/project.py

```diff
@@ -258,16 +360,20 @@ def _validate_classification_prediction(classification_labels, model_type, predi
                 raise ValueError("Prediction output label shape and classification_labels shape do not match")
 
     @staticmethod
-    def validate_df(df: pd.DataFrame, input_types) -> pd.DataFrame:
-        if not set(input_types.keys()).issubset(set(df.columns)):
-            missing_columns = set(input_types.keys()) - set(df.columns)
-            raise ValueError(f"Value mentioned in column_types is not available in dataframe: {missing_columns} ")
-
+    def validate_df(df: pd.DataFrame, column_types) -> pd.DataFrame:
+        if set(column_types.values()) < set(df.columns):
```

Comment:
Should there be `column_types.keys()` instead?

also in the same file we're using `issubset` method, for the sake of consistency we can continue using it here